config/SupportedVehicles.xml

```diff
@@ -96,6 +96,33 @@
         <vehicleConfiguration configFileName="FS22_Lizard_Selfmade_Tow_Truck/pickup2017.xml">
             <loadingArea offset="0 0.65 -2.70" width="2.0" height="2.20" length="4.25"/>
             <options noLoadingIfUnfolded="true"/>
+		</vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPX_816/spx816.xml">
+            <loadingArea offset="0.018 1.39 1.352" width="2.195" height="4.48" length="7.78"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_920/rpx920.xml">
+            <loadingArea offset="0.095 1.39 0.816" width="2.195" height="4.48" length="8.662"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_1226/rpx1226.xml">
+            <loadingArea offset="0.059 1.39 -0.657" width="2.195" height="4.48" length="11.831"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPEX_7519/SPEX.xml">
+            <loadingArea offset="-0.02 0.978 -2.661" width="1.991" height="2.642" length="5.268"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_RollandPack/LSEL10006/LSEL10006_Square.xml">
+            <loadingArea offset="0.015 1.137 -0.161" width="2.352" height="5.301" length="9.627"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_RollandPack/LSP9004/LSP9004_Square.xml">
+            <loadingArea offset="0.031 1.137 -0.654" width="2.352" height="5.042" length="8.432"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PerardPack/platosol/platosol.xml">
+            <loadingArea offset="-0.008 0.923 -0.127" width="2.109" height="3.676" length="5.651"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PerardPack/rpe26/rpe26.xml">
```

Comment:
This one looks just about perfect. Only suggestion I have is add option: ```<options enableRearLoading="true"/>``` if you want to allow rear autoloading via pallet forks.